validator/client/runner.go

```diff
@@ -138,6 +135,9 @@ func run(ctx context.Context, v Validator) {
 
 			// Start fetching domain data for the next epoch.
 			if helpers.IsEpochEnd(slot) {
+				if err := v.LogValidatorGainsAndLosses(slotCtx, slot); err != nil {
```

Comment:
Can we revert this change?  The slot time check is within `LogValidatorGainsAndLosses`

Reply:
Second thought, let's move this after L178